src/components/People/PeopleField/PeopleFields.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/indent */
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../../SearchLink';
+
+type Props = {
+  table:
+    | {
+        value: string;
+        param: string;
+      }
+    | {
+        value: string;
+        param?: undefined;
+      };
+};
```

Comment:
I think shorter syntax for this is
```suggestion
type Props = {
  table: {
        value: string;
        param?: string;
      }
};
```